pkg/resources/kafka/configmap.go

```diff
@@ -254,16 +254,33 @@ func generateListenerSpecificConfig(l *v1beta1.ListenersConfig, log logr.Logger)
 func (r Reconciler) generateBrokerConfig(id int32, brokerConfig *v1beta1.BrokerConfig, extListenerStatuses,
 	intListenerStatuses, controllerIntListenerStatuses map[string]v1beta1.ListenerStatusList,
 	serverPass, clientPass string, superUsers []string, log logr.Logger) string {
+	finalBrokerConfig := getFinalBrokerConfig(id, r.KafkaCluster, log)
```

Comment:
Just one nit. I would name this function and variable differently since it produces the merged readOnlyConfig. I would name it as getReadOnlyConfig or getBrokerReadOnlyConfig.

Reply:
Fixed